src/System.Buffers.Primitives/System/Buffers/OwnedArray.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace System.Buffers
 {
-    public sealed class OwnedArray<T> : OwnedMemory<T>
+    internal sealed class OwnedArray<T> : OwnedBuffer<T>
```

Comment:
There are no new APIs added to OwnedArray and so no need for the type to be public.